src/components/CheckboxWithLabel.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {View, TouchableOpacity} from 'react-native';
+import {View} from 'react-native';
 import _ from 'underscore';
 import styles from '../styles/styles';
 import Checkbox from './Checkbox';
 import Text from './Text';
 import FormHelpMessage from './FormHelpMessage';
 import variables from '../styles/variables';
+import * as Pressables from './Pressable';
```

Comment:
`import {PressableWithoutFeedback} from './Pressable';`

Reply:
Sorry this could cause errors, can we have better export here

https://github.com/Expensify/App/blob/2266274a137b0e2f9c1f8d937ff927626264c77c/src/components/Pressable/index.js#L1-L3

Reply:
I'm not a fan of changing the export, as It will need changes inside the file which imports them already. 
I changed it though to:
```javascript
import * as Pressables from './Pressables`;

const PressableWithFeedback = Pressables.PressableWithFeedback;

...

return (
  <PressableWithFeedback>
    {...}
  </PressableWithFeedback>
);
```

The second option is to use dot notation inside render:
```javascript
<Pressables.PressableWithFeedback>
{...}
</Pressables.PressableWithFeedabck>
```

Reply:
As of now it's used only in one other place in Switch.js I think it's better for changing the exports as it doesn't require many changes now. Help us avoid redundant declarations like this in future other files.

Reply:
Few more PR's with migration are rolling though, would it be OK to just import PressableWithFeedback from the file itself: `import PressableWithFeedback from './Pressable/PressableWithFeedback'` ? It's done in one file like this (button.js) as any tof my tries of modifying this export still casues error. Then we can rid of the index exports too

Reply:
Sounds good to me